src/fides/api/util/tcf/tcf_experience_contents.py

```diff
@@ -523,6 +564,11 @@ def get_tcf_contents(
         matching_privacy_declaration_query=matching_privacy_declarations,
     )
 
+    populate_vendor_relationships_basic_attributes(
+        vendor_map=updated_vendor_relationships_map,
+        matching_privacy_declarations=matching_privacy_declarations,
+    )
```

Comment:
Nice Adam, this was a faster way forward. I was thinking of updating `build_purpose_or_feature_section_and_update_vendor_map` to add special handling for the vendor relationships section but that would add bloat to an already confusing method to parse.  I think this extra loop through the privacy declarations makes sense for readability.

Reply:
yup, i think there were a few ways to skin the cat, probably pros/cons to each of them. glad what i arrived at reads easy enough and seems to get the job done, we can/should come back to it for performance optimizations or refactoring generally!